pkg/datapath/types/node.go

```diff
@@ -42,21 +42,6 @@ type LocalNodeConfiguration struct {
 	// subsequent calls to NodeConfigurationChanged().
 	EnableIPv6 bool
 
-	// UseSingleClusterRoute enables the use of a single cluster-wide route
-	// to direct traffic from the host into the Cilium datapath.  This
-	// avoids the requirement to install a separate route for each node
-	// CIDR and can thus improve the overhead when operating large clusters
-	// with significant node event churn due to auto-scaling.
-	//
-	// Use of UseSingleClusterRoute must be compatible with
-	// EnableAutoDirectRouting. When both are enabled, any direct node
-	// route must take precedence over the cluster-wide route as per LPM
-	// routing definition.
```

Comment:
Fun fact: This comment here is another indication that the flag has not been used (or at least not been used as documented) in years: EnableAutoDirectRouting _cannot_ be enabled together with UseSingleClusterRoute. The former cannot be enabled in tunnel mode, where as the latter required tunnel mode.